packages/tx-construction/test/tx-builder/TxBuilder.bootstrap.test.ts

```diff
@@ -5,71 +5,76 @@ import { SodiumBip32Ed25519 } from '@cardano-sdk/crypto';
 import { dummyLogger } from 'ts-log';
 import { logger, mockProviders as mocks } from '@cardano-sdk/util-dev';
 
-describe('TxBuilder bootstrap', () => {
-  it('awaits for non-empty knownAddresses', async () => {
-    // Initialize the TxBuilder
-    const output = mocks.utxo[0][1];
-    const rewardAccount = mocks.rewardAccount;
-    const knownAddresses = [
-      {
-        accountIndex: 0,
-        address: mocks.utxo[0][1].address,
-        index: 0,
-        networkId: Cardano.NetworkId.Testnet,
-        rewardAccount,
-        type: AddressType.External
-      }
-    ];
-    const inputResolver: Cardano.InputResolver = {
-      resolveInput: async (txIn) =>
-        mocks.utxo.find(
-          ([hydratedTxIn]) => txIn.txId === hydratedTxIn.txId && txIn.index === hydratedTxIn.index
-        )?.[1] || null
-    };
-    const keyAgent = util.createAsyncKeyAgent(
-      await InMemoryKeyAgent.fromBip39MnemonicWords(
+const testTxBuilder = (name: string, useBip32Account: boolean) => {
```

Comment:
Just nitpicking, but you could use [jest.each](https://jestjs.io/blog/2018/05/29/jest-23-blazing-fast-delightful-testing#jest-each) to execute the same test with different parameters.
[An example from our tests](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/projection/test/operators/Mappers/certificates/withStakeKeyRegistrations.test.ts#L9).

Reply:
This is just an fyi. You can resolve the comment without any changes 😄

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1111/commits/d20212380160c89200a2a6368d60943f3a8d3f9c